com.chenyifaer.FaForever.yml

```diff
@@ -0,0 +1,99 @@
+    buildsystem: cmake-ninja
+    config-opts:
+      # taken from https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/5a241420c2ee85cc5080dd7cf7ba3049fd80fcf1/elements/world/webkit2gtk-4.0.bst
+      - -DPORT=GTK
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_WEBDRIVER=OFF
+      - -DUSE_GTK4=OFF
+      - -DUSE_LIBBACKTRACE=OFF
+      - -DUSE_SOUP2=ON
+    modules:
+      - shared-modules/libsoup/libsoup-2.4.json
+
+      - name: unifdef
+        no-autogen: true
+        make-install-args:
+          - prefix=${FLATPAK_DEST}
+        sources:
+          - type: archive
+            url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+            sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+        cleanup:
+          - '*'
+
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  # for some controller mods (see https://github.com/isXander/Controlify/issues/31)
+  - shared-modules/libusb/libusb.json
+
+  - name: faforever
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - bsdtar -Oxf fa-forever.deb data.tar.gz | bsdtar -xf -
+      - install -Dm755 usr/bin/fa-forever /app/bin/faforever
+      - install -Dm644 com.chenyifaer.FaForever.desktop /app/share/applications/com.chenyifaer.FaForever.desktop
+      - install -Dm644 usr/share/icons/hicolor/128x128/apps/fa-forever.png /app/share/icons/hicolor/128x128/apps/com.chenyifaer.FaForever.png
+      - install -Dm644 usr/share/icons/hicolor/32x32/apps/fa-forever.png /app/share/icons/hicolor/32x32/apps/com.chenyifaer.FaForever.png
+      - install -Dm644 usr/share/icons/hicolor/256x256@2/apps/fa-forever.png /app/share/icons/hicolor/256x256@2/apps/com.chenyifaer.FaForever.png
+      - install -Dm644 com.chenyifaer.FaForever.appdata.xml /app/share/appdata/com.chenyifaer.FaForever.appdata.xml
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/cyf/faforever/releases/download/v0.0.1+24/fa-forever_0.0.1+24_amd64.deb
+        dest-filename: fa-forever.deb
+        sha256: 73a741b70e64c67090ac5fca860b306526cbca2dd1c913c9a7ad5efec35e7b12
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/cyf/faforever/releases/latest
+          version-query: .tag_name | sub("v";"")
+          url-query: .assets[] | select(.name=="fa-forever_" + $version + "_amd64.deb") | .browser_download_url
+
+      - type: file
+        url: https://raw.githubusercontent.com/cyf/faforever/v0.0.1+24/linux/com.chenyifaer.FaForever.desktop
+        dest-filename: com.chenyifaer.FaForever.desktop
+        sha256: eeb8e140a8b78911dd22c3a09dde11cfd0ff3d14c1bc0ab2732ca05c0acb78fb
+
+      - type: file
+        url: https://raw.githubusercontent.com/cyf/faforever/v0.0.1+24/linux/com.chenyifaer.FaForever.appdata.xml
+        dest-filename: com.chenyifaer.FaForever.appdata.xml
+        sha256: c6b985d6acb05a13164a7a727b5725da7b72f049dc8ef2fdb5325c8d84d5f418
```

Comment:
Screenshot links are 404

Reply:
Yes, I forgot to upload the screenshot, I am working on it